packages/big-design/src/components/Tabs/Tabs.tsx

```diff
@@ -32,6 +32,8 @@ export const Tabs: React.FC<TabsProps> = memo(
           {items.map(({ id, title, disabled }) => (
             <StyledTab
               activeTab={activeTab}
+              aria-expanded={id === activeTab ? 'true' : 'false'}
+              aria-controls={`${id}-content`}
```

Comment:
I would default it but also accept a template.

Reply:
Do you think this should be a required prop? I kinda don't want to force people to use it if there's a default behavior, but if we don't make it required, some people might not match the ids correctly

Reply:
Thinking about adding `ariaControls?: string;` to the `TabItem` interface

Reply:
Yeah, definitely optional.